stl/inc/sstream

```diff
@@ -373,7 +521,11 @@ private:
 };
 
 template <class _Elem, class _Traits, class _Alloc>
-void swap(basic_stringbuf<_Elem, _Traits, _Alloc>& _Left, basic_stringbuf<_Elem, _Traits, _Alloc>& _Right) {
+void swap(basic_stringbuf<_Elem, _Traits, _Alloc>& _Left, basic_stringbuf<_Elem, _Traits, _Alloc>& _Right)
+#if _HAS_CXX20
+    noexcept(noexcept(_Left.swap(_Right)))
+#endif // _HAS_CXX20

```

Comment:
Similar comment about using `noexcept(condition)` for all Standard modes if this is correct (which I assume it is for `swap`).